cycledash/runs.py

```diff
@@ -21,9 +21,15 @@ class CollisionError(Exception):
 def get_vcf(vcf_id):
     """Return a vcf with a given ID, and the spec and list of contigs for that
     run for use by the /examine page."""
-    with tables(db, 'vcfs') as (con, vcfs):
-        q = select(vcfs.c).where(vcfs.c.id == vcf_id)
-        vcf = dict(con.execute(q).fetchone())
+    projects = cycledash.projects.get_projects_tree_dict()
+    vcf = None
+    for project in projects:
```

Comment:
Good catch. I've updated the references. BAMs are displaying in dalliance once again!